fudge-ratio/src/main/webapp/about/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
     <head>
         <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="about_style.css" />
+        <link rel="stylesheet" href="about.css" />
```

Comment:
I THINK (OPINION), naming something ``x_style.css`` could be a little repetitive, as ``.css`` indicate that this is a style sheet, if I found ``x_code.css`` I will be really scared :v

Reply:
I agree